src/summary/history/notes-card.component.tsx

```diff
@@ -0,0 +1,110 @@
+
+export default function NotesCard(props: NotesCardProps) {
+  const [patientNotes, setPatientNotes] = React.useState(null);
+  const [showMore, setShowMore] = React.useState(false);
+
+  React.useEffect(() => {
+    const abortController = new AbortController();
+
+    encounterResource(props.currentPatient.identifier[0].value, abortController)
+      .then(({ data }) => {
+        if (data.total > 0) {
+          setPatientNotes(getNotes(data.entry));
+        }
+      })
+      .catch(error => {
+        createErrorHandler();
+      });
+    return () => abortController.abort();
+  }, [props.currentPatient.identifier[0].value]);
+
+  return (
+    <SummaryCard name="Notes" match={props.match} styles={{ width: "45.5rem" }}>
+      <SummaryCardRow>
+        <SummaryCardRowContent justifyContent="space-between">
+          <NotesValues
+            date="Date"
+            dateStyles={{ color: "var(--omrs-color-ink-medium-contrast)" }}
+            label="Encounter type,"
+            labelStyles={{ color: "var(--omrs-color-ink-medium-contrast)" }}
+            location=" Location"
+            locationStyles={{ color: "var(--omrs-color-ink-medium-contrast)" }}
+            author="Author"
+            authorStyles={{ color: "var(--omrs-color-ink-medium-contrast)" }}
+          />
+        </SummaryCardRowContent>
+      </SummaryCardRow>
+      {patientNotes &&
+        patientNotes.slice(0, showMore ? patientNotes.length : 5).map(note => {
+          return (
+            <SummaryCardRow key={note.id} linkTo="/notes">
+              <NotesValues
+                date={convertdate(note.location[0].period.end)}
+                dateClassName="omrs-bold"
+                dateStyles={{ alignContent: "left" }}
+                label={note.type[0].coding[0].display}
+                labelClassName="omrs-bold"
+                location={note.location[0].location.display}
+                locationStyles={{
+                  color: "var(--omrs-color-ink-medium-contrast)"
+                }}
+                author={getAuthorName(note.extension)}
+                authorClassName="omrs-bold"
+              />
+            </SummaryCardRow>
+          );
+        })}
+      <SummaryCardFooter linkTo="/notes">
+        <div className={style.moreNotes}>
+          <svg className="omrs-icon">
+            <use
+              xlinkHref="#omrs-icon-chevron-right"
+              fill="var(--omrs-color-ink-low-contrast)"
+            ></use>
+          </svg>
+          <p>See all</p>
+        </div>
+      </SummaryCardFooter>
+    </SummaryCard>
+  );
+}
+function getNotes(notes) {
+  return notes.map(note => note.resource);
+}
+function getAuthorName(extension: any): string {
+  let authorName = "";
+  extension.filter(function(author) {
+    if (author.url === "changedBy") {
+      authorName = author.valueString;
+    }
+  });
+  return authorName.toUpperCase();
+}
+function convertdate(date: string): string {

```

Comment:
Javascript functions should be camel cased:

```suggestion
function convertDate(date: string): string {
```